templates/trivy-step.tmpl

```diff
@@ -0,0 +1,12 @@
+steps:
+- command: ls
+  plugins:
+    - equinixmetal-buildkite/trivy#{{ .TrivyPlugin }}
+	    severity: {{ .Severity }}
+	    ignore-unfixed: {{ .IgnoreUnfixed }}
+	    security-checks: {{ join .SecurityChecks }}
+	    skip-files: '{{ .SkipFiles }}'
+- label: ":sparkles: SHELL CHECK"
+  plugins:
+    - shellcheck#{{ .ShellPlugin }}:
+        files: script.sh
```

Comment:
Instead of putting the template in a separate file, you could make it a global constant string in one of the Go source files. That way you do not need to handle a scenario where the file could not be read.

Reply:
Currently the requirement is only for trivy-step template  however we might need to add few more templates like helm-lint etc for the same dynamic pipeline in future.So, In my opinion having separate templates files would be better approach.

Reply:
Spoke with Hari about this. It sounds like the intention is to run this in a container image where `templates/` is guaranteed to exist relative to the current working directory of a program that uses this library.

Reply:
Thanks @stephen-fox!